app/Listeners/StoreUserRegistered.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Listeners;
+
+use App\Events\UserRegistered;
+use App\Mail\EmailConfirm;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Support\Facades\Mail;
+
+class StoreUserRegistered
```

Comment:
название тут не оч подходит
лучше так
```suggestion
class SendConfirmEmail
```